packages/bot-web-ui/src/components/quick-strategy/inputs/qs-input-label.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { Popover, Text } from '@deriv/components';
+import { useStore } from '@deriv/stores';
+
+type TQSInputLabel = {
+    children?: React.ReactNode;
+    label?: string;
+    description?: string;
+    fullwidth?: boolean;
+};
+
+const QSInputLabel: React.FC<TQSInputLabel> = ({ label, description, fullwidth = false }) => {
```

Comment:
@sandeep-deriv Bhai can we wrap this component with `observer` as it is using mobx store observable?

Reply:
sure 👍